From penberg@cs.helsinki.fi  Mon Aug 17 12:58:36 2009
From: Pekka Enberg <penberg@cs.helsinki.fi>
Date: Wed, 12 Aug 2009 11:03:35 +0300
Subject: Staging: w35und: Cleanup the mlmetxrx_f.h header file
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org, Pekka Enberg <penberg@cs.helsinki.fi>, Pavel Machek <pavel@ucw.cz>
Message-ID: <1250064219-7372-3-git-send-email-penberg@cs.helsinki.fi>


This patch removes the unused MLMEProcThread and MLMEReturnPacket
declarations and indents the header file.

Acked-by: Pavel Machek <pavel@ucw.cz>
Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/winbond/mlmetxrx_f.h |   24 ++++++------------------
 1 file changed, 6 insertions(+), 18 deletions(-)

--- a/drivers/staging/winbond/mlmetxrx_f.h
+++ b/drivers/staging/winbond/mlmetxrx_f.h
@@ -10,28 +10,16 @@
 
 #include "core.h"
 
-void
-MLMEProcThread(
-     struct wbsoft_priv *    adapter
-	);
-
-void MLME_GetNextPacket(  struct wbsoft_priv * adapter,  PDESCRIPTOR pDes );
-u8 MLMESendFrame( struct wbsoft_priv * adapter,
-					u8	*pMMPDU,
-					u16	len,
-					 u8	DataType);
+void MLME_GetNextPacket(struct wbsoft_priv *adapter, PDESCRIPTOR pDes);
+u8 MLMESendFrame(struct wbsoft_priv *adapter,
+		 u8 * pMMPDU, u16 len, u8 DataType);
 
 void
-MLME_SendComplete(  struct wbsoft_priv * adapter,  u8 PacketID,  unsigned char SendOK );
+MLME_SendComplete(struct wbsoft_priv *adapter, u8 PacketID,
+		  unsigned char SendOK);
 
-void
-MLMEReturnPacket(
-     struct wbsoft_priv *    adapter,
-     u8 *          pRxBufer
-   );
 #ifdef _IBSS_BEACON_SEQ_STICK_
-s8 SendBCNullData(struct wbsoft_priv * adapter, u16 wIdx);
+s8 SendBCNullData(struct wbsoft_priv *adapter, u16 wIdx);
 #endif
 
 #endif
-
